client/src/modules/Schedule/pages/SchedulePage/index.tsx

```diff
@@ -23,6 +25,7 @@ export function SchedulePage(props: PageProps) {
   const [cipher, setCipher] = useState('');
   const [courseTask, setCourseTask] = useState<null | Record<string, any>>(null);
   const [copyModal, setCopyModal] = useState<{ id?: number } | null>(null);
+  const [selectedTab, setSelectedTab] = useState<string>(ALL_TAB_KEY);
```

Comment:
`statusFilter` at `<TableView />` used `useLocalStorage` hook. I move it from component level to page level and changed to `useState`. Should the filtering via tabs also use `useLocalStorage`?

Reply:
I would say - yes

Reply:
Ok. I will fix and update the PR

Reply:
Fixed, please review